com.helix_editor.Helix.yml

```diff
@@ -0,0 +1,86 @@
+app-id: com.helix_editor.Helix
+
+runtime: org.freedesktop.Sdk
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+command: hx
+
+modules:
+  - name: hx
+    buildsystem: simple
+    build-commands:
+      - install -D hx /app/bin/hx
+
+      - mkdir -p /app/lib/helix
+      - rm -rf /app/lib/helix/runtime
+      - cp -r runtime /app/lib/helix/runtime
+
+      - install -D Helix.desktop /app/share/applications/com.helix_editor.Helix.desktop
+      - install -D helix.png /app/share/icons/hicolor/256x256/apps/com.helix_editor.Helix.png
+
+      - install -D helix.appdata.xml /app/share/metainfo/com.helix_editor.Helix.metainfo.xml
```

Comment:
these files should be submitted upstream. A desktop app without icon or .desktop is just a bug.

Reply:
Icon and .desktop are from upstream (https://github.com/helix-editor/helix/tree/master/contrib), but they don't put them in release archive. The appdata manifest however I wrote myself